tests/integration/test_fetch_orderbook.py

```diff
@@ -41,48 +42,90 @@ def test_get_order_rewards(self):
         self.assertIsNone(pd.testing.assert_frame_equal(expected, rewards_df))
 
     def test_get_batch_rewards(self):
-        block_number = 16735007
-        block_range = BlockRange(block_number, block_number + 50)
+        block_number = 16846500
```

Comment:
So, the starting  block seems to be 16,846,500
while on line 51, I see  block  16,846,495